src/braket/flake8_plugins/braket_checkstyle_plugin.py

```diff
@@ -53,9 +54,9 @@ class _Visitor(ast.NodeVisitor):
     MISC_REGEX = re.compile(
         r"^(\s*)(Throws|Raises|See Also|Note|Example|Examples|Warnings)\s*:\s*$"
     )
-    ARG_INFO_REGEX = re.compile(r"^(\s*)(\*{0,2}\w*)\s*(\([^:]*\))?\s*:\s*(.*)")
+    ARG_INFO_REGEX = re.compile(r"^(\s*)`{0,2}(\*{0,2}\w*)`{0,2}\s*(\([^:]*\))?\s*:\s*(.*)")
```

Comment:
This is what we have currently`re.compile(r"^(\s*)(`?\*{0,2}\w*`?)\s*(\([^:]*\))?\s*:\s*(.*)")`, does your change affects the grouping from what we have currently in the logic [here](https://github.com/aws/amazon-braket-build-tools/blob/main/src/braket/flake8_plugins/braket_checkstyle_plugin.py#L190-L193)?

Reply:
I'm not sure what you mean by the first part of it, but the new additions are not captured so they shouldn't affect the captured groups. Also, the unit tests are passing.

Reply:
My mistake - I didn't pull before I made the branch. However, I don't think it's correct to include the ` mark in the capture and then code for removing it. I've modified the regex.